java/impl/gcp/gcp.iml

```diff
@@ -13,7 +13,7 @@
     </content>
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="module" module-name="psoxy-core" />
+    <orderEntry type="module" module-name="core" />
```

Comment:
I think this is the main issue we have with *.iml files; maye we have configured the module is *core* in yours and *gateway-core* in mine and that could cause the mismatch between files in our commits